tests/test_fips.py

```diff
@@ -350,3 +367,23 @@ def test_nss_firefox_cert(container_per_test: ContainerData) -> None:
     c.check_output(
         'certutil -R -k rsa -g 2048 -s "CN=Daniel Duesentrieb3,O=Example Corp,L=Mountain View,ST=California,C=DE" -d "${PWD}/nssdb" -o cert9.cer -f password.txt -z seedfile.dat',
     )
+
+
+@pytest.mark.parametrize(
+    "container",
+    FIPS_OPENJDK_IMAGES,
+    indirect=True,
+)
+def test_openjdk_cert(container: ContainerData) -> None:
+    c = container.connection
+
+    # Clone, compile and run the scripts to check the FIPS provider
```

Comment:
This line is not really that helpful tbh
```suggestion
```

Reply:
Removed